kebbie/cmd.py

```diff
@@ -148,3 +170,11 @@ def cli():
         for c in correctors:
             c.emulator.show_keyboards()
             print(f"Predictions : {c.emulator.get_predictions()}")
+
+    elif args.cmd == "get_page_source":
+        correctors = instantiate_correctors(args.keyboard)
+        for c in correctors:
+            page_source = c.emulator.get_page_source(print_page_source=args.print_page_source)
```

Comment:
Adding a method just for this is overkill IMO.

Just do : 

```suggestion
            page_source = c.emulator.driver.page_source
            
            if args.print_page_source:
                print(page_source)
```